e2e/content-services/upload/upload-dialog.ts

```diff
@@ -78,14 +80,15 @@ describe('Upload component', () => {
     });
 
     afterEach(async (done) => {
-        const nodesPromise = await contentServicesPage.getElementsDisplayedId();
-
-        nodesPromise.forEach(async (currentNodePromise) => {
-            await currentNodePromise.then(async (currentNode) => {
-                if (currentNode && currentNode !== 'Node id') {
-                    await uploadActions.deleteFileOrFolder(currentNode);
-                }
-            });
+        contentServicesPage.numberOfResultsDisplayed().then(async (nbResults) => {
```

Comment:
I think you could try rewriting this line as
`const nbResults = async contentServicesPage.numberOfResultsDisplayed();`
to remove the "then"